src/components/widgets/CheckBox.js

```diff
@@ -30,11 +30,17 @@ class CheckBox extends PureComponent {
     }
 
     const edit = (
-      <Checkbox
-        {...input}
-        checked={checkedStatus}
-        onChange={(e) => this.handleToggle(e, checkBoxValue)}
-      >{title}</Checkbox>
+      <span>
+        <span>{title}</span>
```

Comment:
@rhenshaw56 I don't think you committed all the files, this is not working. I'm guessing you made some CSS changes that are not here.

Reply:
@jomadoye that's right, I added the styles [here](https://github.com/openmrs/openmrs-owa-labworkflow/pull/62)